xmodule/vertical_block.py

```diff
@@ -250,6 +250,14 @@ def definition_from_xml(cls, xml_object, system):
         children = []
         for child in xml_object:
             try:
+                if is_pointer_tag(child):
+                    id_generator = system.id_generator
+                    url_name = child.get('url_name')
+                    block_type = child.tag
+                    def_id = id_generator.create_definition(block_type, url_name)
+                    child, _ = cls.load_definition_xml(child, system, def_id)
+                    child.set('url_name', url_name)
```

Comment:
```suggestion
                    child, _ = cls.load_definition_xml(child, system, def_id)
                    # The pointed-to child may or may not set its url_name. Either way, we ensure
                    # it is set here because there is parsing code later that expects it.
                    child.set('url_name', url_name)
```